locale/ru-ru/dialog/daily/daily-wind-light-local.dialog

```diff
@@ -1,3 +1,2 @@
-# auto translated from en-us to ru-ru
-Будет легкий ветер, идущий от {direction} {day} в {speed} {speed_unit}.
-Будет не очень ветрено {day}
+{day} будет легкий {direction} ветер, {speed} {speed_unit}
+{day} будет не очень ветрено
```

Comment:
while the missing location in some of the dialogs is not a big deal (as you are specifically asking for a location and is more of a confirmation), do you want to omit the speed, speed_unit and direction here?
50-50 chance that cant be influenced.

Reply:
It is the same in the `en-us` translation, so I kept it as is. There should probably be some general style guide for all languages.